scripts/actions/spells/trust/koru-moru.lua

```diff
@@ -19,26 +19,32 @@ spellObject.onMobSpawn = function(mob)
         [xi.magic.spell.AJIDO_MARUJIDO] = xi.trust.messageOffset.TEAMWORK_2,
     })
 
-    mob:addSimpleGambit(ai.t.SELF, ai.c.MPP_LT, 5, ai.r.JA, ai.s.SPECIFIC, xi.ja.CONVERT)
+    mob:addGambit(ai.t.SELF, { ai.c.MPP_LT, 5 }, { ai.r.JA, ai.s.SPECIFIC, xi.ja.CONVERT })
 
-    mob:addSimpleGambit(ai.t.PARTY, ai.c.HPP_LT, 50, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.CURE)
+    mob:addGambit(ai.t.PARTY, { ai.c.HPP_LT, 50 }, { ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.CURE })
 
-    mob:addSimpleGambit(ai.t.MELEE, ai.c.NOT_STATUS, xi.effect.HASTE, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.HASTE)
-    mob:addSimpleGambit(ai.t.CASTER, ai.c.NOT_STATUS, xi.effect.REFRESH, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.REFRESH)
-    mob:addSimpleGambit(ai.t.TANK, ai.c.NOT_STATUS, xi.effect.REFRESH, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.REFRESH)
-    mob:addSimpleGambit(ai.t.RANGED, ai.c.NOT_STATUS, xi.effect.FLURRY_II, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.FLURRY) -- xi.effect.FLURRY_II is not a typo
+    mob:addGambit(ai.t.MELEE, { ai.c.NOT_STATUS, xi.effect.HASTE }, { ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.HASTE })
+    mob:addGambit(ai.t.CASTER, {
+        { ai.c.NOT_STATUS, xi.effect.REFRESH },
+        { ai.c.NOT_STATUS, xi.effect.SUBLIMATION_ACTIVATED },
+        { ai.c.NOT_STATUS, xi.effect.SUBLIMATION_COMPLETE },
+    }, { ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.REFRESH })
+    mob:addGambit(ai.t.TANK, { ai.c.NOT_STATUS, xi.effect.REFRESH }, { ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.REFRESH })
+    mob:addGambit(ai.t.RANGED, {
+            { ai.c.NOT_STATUS, xi.effect.FLURRY_II },
+            { ai.c.NOT_STATUS, xi.effect.HASTE }, -- No overwriting Haste
+        }, { ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.FLURRY }) -- xi.effect.FLURRY_II is not a typo
```

Comment:
This comment probably needs to be moved up to the line where it actually has the flurry_II effect